### PR TITLE
Completed Binary Search-2

### PR DESCRIPTION
**Pull request by sh-ruchi**

### CODE REVIEW

File: FirstLastPos.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// Time Complexity : 2 O(LOG N)
```
Comment (daredevilteja) [inline FirstLastPos.java:1]: what is n?

File: MinElementInRotatedArray.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+// Time Complexity : O(Log n)
```
Comment (daredevilteja) [inline MinElementInRotatedArray.java:1]: what is n

File: SearchInSorterArrUnknownSize.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// Time Compexity : O(log n)
```
Comment (daredevilteja) [inline SearchInSorterArrUnknownSize.java]: what is n?

**Review by daredevilteja — CHANGES_REQUESTED** [review]

**Comment by daredevilteja** [comment]
Please mention whether it ran successfully on leetcode or not, for every problem.

File: SearchInSorterArrUnknownSize.java (hunk 3 above)
Comment (sharma-ruchika) [reply SearchInSorterArrUnknownSize.java]: n is the length of nums array .

**Comment by sharma-ruchika** [comment]
Its mentioned at the top  line 3 , that it ran successfully on Leetcode.

File: FirstLastPos.java (hunk 1 above)
Comment (sharma-ruchika) [reply FirstLastPos.java:1]: n is length of array

**Comment by daredevilteja** [comment]
> Its mentioned at the top line 3 , that it ran successfully on Leetcode.

You mentioned it only for one problem, you have to mention it for all the problems

**Pull request closed without merging**